src/index.tsx

```diff
@@ -1,10 +1,49 @@
 import ReactDOM from 'react-dom';
-import { HashRouter } from 'react-router-dom';
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
 import { App } from './App';
+import { Tabs } from './components/Tabs';
+import { NotFound } from './components/NotFound';
+import { NotFoundRedirect } from './components/NotFoundRedirect';
+import { Home } from './components/Home';
+import { Navigate } from './components/Navigate';
+
+const router = createBrowserRouter([
+  {
+    path: '/home',
+    element: <Navigate />,
+  },
+  {
+    path: '/',
+    element: <App />,
```

Comment:
Not correct logic.
Your application starting point should be `App`.
Inside the app, you will render `AppRouter`.
For wrapping your router in some layout you can create a separate component `Layout` where you will use `<Outlet/>`